### PR TITLE
Update rekor release to 1.0 prerelease

### PR DESCRIPTION
**Pull request by priyawadhwa**
Signed-off-by: Priya Wadhwa <priya@chainguard.dev>

## Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/rekor/Chart.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@ description: Part of the sigstore project, Rekor is a timestamping server and tr
 
 type: application
 
-version: 0.3.21
-appVersion: 0.12.2
+version: 0.3.22
```
Comment (haydentherapper) [inline charts/rekor/Chart.yaml:7]: Should we bump the version to 0.9.9 to signify a prerelease?

Comment (haydentherapper) [reply charts/rekor/Chart.yaml:7]: Just keeping adding 9s if we have more prereleases :)

Comment (hectorj2f) [reply charts/rekor/Chart.yaml:7]: I had a look and I believe we could create a Chart with `version: 1.0.0-rc.0`, cc @cpanato

Comment (cpanato) [reply charts/rekor/Chart.yaml:7]: yes we should be able to do that

Comment (priyawadhwa) [reply charts/rekor/Chart.yaml]: thanks, updated!

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**